src/policy/truc_policy.h

```diff
@@ -15,11 +15,12 @@
 #include <set>
```

Comment:
In f5b26e5b9be41cdab6e75aa0a3fbbb765a96ed06
Here are a couple of places that were not replaced
```diff
diff --git a/src/policy/truc_policy.h b/src/policy/truc_policy.h
index d1e65806cc0..37a445bcc14 100644
--- a/src/policy/truc_policy.h
+++ b/src/policy/truc_policy.h
@@ -23,18 +23,18 @@ static constexpr decltype(CTransaction::version) TRUC_VERSION{3};
 // of 2 and ancestor set size of 2.
 /** Maximum number of transactions including an unconfirmed tx and its descendants. */
 static constexpr unsigned int TRUC_DESCENDANT_LIMIT{2};
-/** Maximum number of transactions including a V3 tx and all its mempool ancestors. */
+/** Maximum number of transactions including a TRUC tx and all its mempool ancestors. */
 static constexpr unsigned int TRUC_ANCESTOR_LIMIT{2};
 
-/** Maximum sigop-adjusted virtual size of all v3 transactions. */
+/** Maximum sigop-adjusted virtual size of all TRUC transactions. */
 static constexpr int64_t TRUC_MAX_VSIZE{10000};
-/** Maximum sigop-adjusted virtual size of a tx which spends from an unconfirmed v3 
transaction. */
+/** Maximum sigop-adjusted virtual size of a tx which spends from an unconfirmed TRU
C transaction. */
 static constexpr int64_t TRUC_CHILD_MAX_VSIZE{1000};
 // These limits are within the default ancestor/descendant limits.
 static_assert(TRUC_MAX_VSIZE + TRUC_CHILD_MAX_VSIZE <= DEFAULT_ANCESTOR_SIZE_LIMIT_K
VB * 1000);
 static_assert(TRUC_MAX_VSIZE + TRUC_CHILD_MAX_VSIZE <= DEFAULT_DESCENDANT_SIZE_LIMIT
_KVB * 1000);
 
-/** Must be called for every transaction, even if not v3. Not strictly necessary for
 transactions
+/** Must be called for every transaction, even if not TRUC. Not strictly necessary f
or transactions
  * accepted through AcceptMultipleTransactions.
  *
  * Checks the following rules:
diff --git a/src/test/fuzz/package_eval.cpp b/src/test/fuzz/package_eval.cpp
index ecacfed58ed..71f72847dd4 100644
--- a/src/test/fuzz/package_eval.cpp
+++ b/src/test/fuzz/package_eval.cpp
@@ -225,7 +225,7 @@ FUZZ_TARGET(tx_package_eval, .init = initialize_tx_pool)
                     tx_mut.vin.emplace_back();
                 }
 
-                // Make a p2pk output to make sigops adjusted vsize to violate v3, p
otentially, which is never spent
+                // Make a p2pk output to make sigops adjusted vsize to violate TRUC,
 potentially, which is never spent
                 if (last_tx && amount_in > 1000 && fuzzed_data_provider.ConsumeBool(
)) {
                     tx_mut.vout.emplace_back(1000, CScript() << std::vector<unsigned
 char>(33, 0x02) << OP_CHECKSIG);
                     // Don't add any other outputs.
diff --git a/src/test/util/txmempool.cpp b/src/test/util/txmempool.cpp
index 1153df4ade7..37cd0de5d2a 100644
--- a/src/test/util/txmempool.cpp
+++ b/src/test/util/txmempool.cpp
@@ -150,7 +150,7 @@ void CheckMempoolV3Invariants(const CTxMemPool& tx_pool)
             // Check that special maximum virtual size is respected
             Assert(entry.GetTxSize() <= TRUC_MAX_VSIZE);
 
-            // Check that special v3 ancestor/descendant limits and rules are always
 respected
+            // Check that special TRUC ancestor/descendant limits and rules are alwa
ys respected
             Assert(entry.GetCountWithDescendants() <= TRUC_DESCENDANT_LIMIT);
             Assert(entry.GetCountWithAncestors() <= TRUC_ANCESTOR_LIMIT);
             Assert(entry.GetSizeWithDescendants() <= TRUC_MAX_VSIZE + TRUC_CHILD_MAX
_VSIZE);
@@ -159,12 +159,12 @@ void CheckMempoolV3Invariants(const CTxMemPool& tx_pool)
             // If this transaction has at least 1 ancestor, it's a "child" and has r
estricted weight.
             if (entry.GetCountWithAncestors() > 1) {
                 Assert(entry.GetTxSize() <= TRUC_CHILD_MAX_VSIZE);
-                // All v3 transactions must only have v3 unconfirmed parents.
+                // All TRUC transactions must only have TRUC unconfirmed parents.
                 const auto& parents = entry.GetMemPoolParentsConst();
                 Assert(parents.begin()->get().GetSharedTx()->version == TRUC_VERSION
);
             }
         } else if (entry.GetCountWithAncestors() > 1) {
-            // All non-v3 transactions must only have non-v3 unconfirmed parents.
+            // All non-TRUC transactions must only have non-TRUC unconfirmed parents
.
             for (const auto& parent : entry.GetMemPoolParentsConst()) {
                 Assert(parent.get().GetSharedTx()->version != TRUC_VERSION);
             }
diff --git a/src/test/util/txmempool.h b/src/test/util/txmempool.h
index 9986cabf7e9..57027cc0e62 100644
--- a/src/test/util/txmempool.h
+++ b/src/test/util/txmempool.h
@@ -47,12 +47,12 @@ std::optional<std::string>  CheckPackageMempoolAcceptResult(const
 Package& txns,
                                                             bool expect_valid,
                                                             const CTxMemPool* mempoo
l);
 
-/** For every transaction in tx_pool, check v3 invariants:
- * - a v3 tx's ancestor count must be within V3_ANCESTOR_LIMIT
- * - a v3 tx's descendant count must be within V3_DESCENDANT_LIMIT
- * - if a v3 tx has ancestors, its sigop-adjusted vsize must be within V3_CHILD_MA
X_VSIZE
- * - any non-v3 tx must only have non-v3 parents
- * - any v3 tx must only have v3 parents
+/** For every transaction in tx_pool, check TRUC invariants:
+ * - a TRUC tx's ancestor count must be within V3_ANCESTOR_LIMIT
+ * - a TRUC tx's descendant count must be within V3_DESCENDANT_LIMIT
+ * - if a TRUC tx has ancestors, its sigop-adjusted vsize must be within V3_CHILD_
MAX_VSIZE
+ * - any non-TRUC tx must only have non-TRUC parents
+ * - any TRUC tx must only have TRUC parents
  *   */
 void CheckMempoolV3Invariants(const CTxMemPool& tx_pool);
 
```